mujoco_warp/_src/io.py

```diff
@@ -363,9 +370,12 @@ def put_model(mjm: mujoco.MjModel) -> types.Model:
   m.geom_aabb = wp.array(mjm.geom_aabb, dtype=wp.vec3, ndim=3)
   m.geom_rbound = wp.array(mjm.geom_rbound, dtype=wp.float32, ndim=1)
   m.geom_dataid = wp.array(mjm.geom_dataid, dtype=wp.int32, ndim=1)
+  m.geom_group = wp.array(mjm.geom_group, dtype=wp.int32, ndim=1)
   m.mesh_vertadr = wp.array(mjm.mesh_vertadr, dtype=wp.int32, ndim=1)
   m.mesh_vertnum = wp.array(mjm.mesh_vertnum, dtype=wp.int32, ndim=1)
   m.mesh_vert = wp.array(mjm.mesh_vert, dtype=wp.vec3, ndim=1)
+  m.mesh_faceadr = wp.array(mjm.mesh_faceadr, dtype=wp.int32, ndim=1)
+  m.mesh_face = wp.array(mjm.mesh_face, dtype=wp.int32, ndim=2)
```

Comment:
since `mesh_face` is `(nmeshface, 3)` this array could be `wp.array(mjm.mesh_face, dtype=wp.vec3i, ndim=1)`